sys/nginx/default.conf.template

```diff
@@ -43,6 +38,11 @@ server {
         expires 365d;
     }
 
+    # local = 127.0.0.11
+    # aws = 169.254.169.253
```

Comment:
sure you want to make this ip public?

Reply:
They are of public knowledge already, the first IP is the local resolve address when running locally. The second IP is a special internal IP used by AWS to provide access to their API from within the instances/container, it also listens on :53 to provide dns resolution.

Reply:
One is an RFC1918 (private addresses) the other is a link-local address .. both are non-routable addresses. Quite safe ;)

Reply:
👍